packages/vats/test/localchain.test.js

```diff
@@ -0,0 +1,156 @@
+import { test as anyTest } from '@agoric/swingset-vat/tools/prepare-test-env-ava.js';
+import { reincarnate } from '@agoric/swingset-liveslots/tools/setup-vat-data.js';
+import { E } from '@endo/far';
+import { M, matches } from '@endo/patterns';
+import { makeDurableZone } from '@agoric/zone/durable.js';
+import { AmountMath, AssetKind, makeIssuerKit } from '@agoric/ertp';
+import { buildRootObject } from '../src/vat-bank.js';
+import { prepareLocalChainTools } from '../src/localchain.js';
+
+/** @type {import('ava').TestFn<ReturnType<makeTestContext>>} */
+const test = anyTest;
+
+const makeTestContext = async t => {
+  const { fakeVomKit } = reincarnate({ relaxDurabilityRules: false });
+  const provideBaggage = key => {
+    const root = fakeVomKit.cm.provideBaggage();
+    const zone = makeDurableZone(root);
+    return zone.mapStore(`${key} baggage`);
+  };
+  const baggage = provideBaggage('localchainTest');
+  const bankVat = E(buildRootObject)(null, null, baggage);
```

Comment:
the bankVat shouldn't share baggage with localChain.

hm. I see `subZone` below. Maybe that's close enough.

but giving each thing that's usually a vat its own baggage with `provideBaggage('bank')` and `provideBaggage('localChain')` seems more straightforward

Reply:
the bridge is normally its own vat too

Reply:
Thank you, updated the context setup to have separate zones and baggage for each of the three